VTEX - Ad Network API.json

```diff
@@ -608,6 +617,399 @@
+                                    },
+                                    "ref": {
+                                        "type": "string",
+                                        "description": "Previous page URL (accessed before the current page URL) or mobile equivalent.",
+                                        "example": "https://www.apiexamples.com.br"
+                                    },
+                                    "sessionId": {
+                                        "type": "string",
+                                        "description": "Session identifier, which expires after 30 minutes. Use UUID Version 4.",
+                                        "example": "fb252d90-7494-4097-90fd-cc73eeaa575c"
+                                    },
+                                    "orderGroup": {
+                                        "type": "string",
+                                        "description": "OMS order group for the created order. The orderGroup should be the same as the one created in the OMS to enable the correlation between the order events and the order tables.",
+                                        "example": "XYZ1241DVD"
+                                    },
+                                    "channel": {
+                                        "type": "string",
+                                        "description": "Specifies if it is a mobile or web client.",
+                                        "enum": [
+                                            "website",
+                                            "android",
+                                            "ios",
+                                            "msite",
+                                            "whatsapp"
+                                        ],
+                                        "example": "website"
+                                    }
+                                }
+                            }
+                        }
+                    }
+                },
+                "responses": {
+                    "200": {
+                        "description": "OK"
+                    }
+                }
+            }
+        },
+        "/banner/query": {
+            "servers": [
+                {
+                    "url": "https://ad-server.vtex.com/api/ad-server/ads/",
+                    "description": "Ad Network server."
+                }
+            ],
+            "post": {
+                "summary": "Get banner information",
+                "description": "Retrieves banner information. Applicable to non-VTEX stores integrating with VTEX Ad Network with targeting.\r\n\r\n## Permissions\r\n\r\nThis endpoint does not require [authentication](https:\/\/developers.vtex.com\/docs\/guides\/authentication) or [permissions](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).",
```

Comment:
In this one you wrote "VTEX Ad Network", but above there are examples of just "Ad Network". Please correct the feature's name in the incorrect places and create a common standard.